tests/testthat/test-tmb-methods.R

```diff
@@ -580,62 +573,46 @@ test_that("model.matrix works as expected with defaults", {
   )
 })
 
-test_that("model.matrix works as expected with includes", {
-  object <- get_mmrm_tmb()
```

Comment:
can we include these tests still as much as possible with the new interface? and also test the `drop_response` thing?